### ISSUE TITLE
Double slash (``/'') should be equivalent to single slash in RPCs

### ISSUE THREAD

**Issue by elric1** [tezedge/tezedge#650]
For example:
```
:; curl -s http://127.0.0.1:8732/chains/main/blocks//head/header
Compat { error: ErrorMessage { msg: "Invalid block_id_param: , reason: missing checksum" } }
:;  curl -s http://127.0.0.1:8732/chains/main/blocks/head/header
{"hash":"BLockGenesisGenesisGenesisGenesisGenesisf79b5d1CoW2","chain_id":"NetXdQprcVkpaWU","level":0,"proto":0,"predecessor":"BLockGenesisGenesisGenesisGenesisGenesisf79b5d1CoW2","timestamp":"2018-06-30T16:07:32Z","validation_pass":0,"operations_hash":"LLoZS2LW3rEi7KYU4ouBQtorua37aWWCtpDmv1n2x3xoKi6sVXLWp","fitness":[],"context":"CoVNUgnfVocNYM2HNkHxMMcZdzqFDkCQ5vEGa3gsdpkidoA1QPZo","protocol":"PrihK96nBAFSxVL1GLJTVhu9YnzkMFiBeuJRPA8NwuZVZCE1L6i"}
```
The former command should likely return the same as the latter command. This would match the behaviour of Nomadic Labs `tezos-node` which would help make sure that scripts that work against NL `tezos-node` would work without update with `tezedge`.

**Comment by tizoc** [comment]
@elric1 We are currently doing some work on the RPCs, I will look into that. Thanks for the report.

**Comment by tizoc** [comment]
This was solved in #653 (quick fix, need to do some research to make sure we interpret paths exactly in the same way as Octez), and will be included in the next release of TezEdge which will be happening soon.
